### ISSUE TITLE
[NFR] Even better first boot experience in headless situations

### ISSUE THREAD

**Issue by le51** [zynthian/zynthian-issue-tracking#1112]
**In some particular conditions, install process leads you to a dead end**
I've tested a lot of fresh install with last week image and so far, process is correct, but 

**To Reproduce**
in this "special" use case
1. You need a freshly burned Zynthian Oram SD card with, no display (headless) or some spi lcd (wich need to be configured through webconf) and no wired network connection
2. Boot on SD card, ..., wait 
3. System looks healthy (or not) but no remote access is enabled, and Zynthian is unusable

**Expected behavior**
at first boot, if no wired connection is established,, start Zynthian in Access Point mode

**Comment by brunetton** [comment]
Indeed, that makes sense. I had the exact same issue and expected to Zynthian to activate an access point on "first" boot when no network connection is configured and no network is available.

This of course doesn't applies to the next boots, as we usually don't want access point to be activated always by default.

This makes me think that a solution for headless use could be a physical trigger (simple on/off button) that could command access point creation *and* (optionally, I presume) VNC access. I'm thinking of trying this for my usage.

**Comment by riban-bw** [comment]
This may be integrated with a safe mode startup as requested in #1433.
